ng-package.json

```diff
@@ -4,15 +4,17 @@
     "entryFile": "src/index.ts",
     "umdModuleIds": {
       "algoliasearch": "algoliasearch",
+      "algoliasearch/lite": "algoliasearch",
       "algoliasearch/index": "algoliasearch",
       "instantsearch.js/es": "instantsearch",
       "instantsearch.js/es/connectors": "instantsearch.connectors",
       "nouislider": "noUiSlider",
       "querystring-es3/encode": "qs.encode",
-      "algoliasearch-helper": "algoliasearchHelper"
-    }
+      "algoliasearch-helper": "algoliasearchHelper",
+      "instantsearch.js/es/widgets/index/index": "instantsearch.widgets.index"
```

Comment:
widgets and connectors should be together on this list, could you sort it alphabetically?

Reply:
you're right it's better: done b9ccc64

Reply:
done here b9ccc64f5625da5cae1d4151bfafb712f76e8b5c